unattended_installer/install_functions/installVariables.sh

```diff
@@ -15,9 +15,6 @@ readonly wazuh_install_vesion="0.1"
 ## Links and paths to resources
 readonly resources="https://${bucket}/${wazuh_major}"
 readonly base_url="https://${bucket}/${repository}"
-base_path="$(dirname "$(readlink -f "$0")")"
-readonly base_path
-config_file="${base_path}/config.yml"
 readonly tar_file_name="wazuh-install-files.tar"
 tar_file="${base_path}/${tar_file_name}"
```

Comment:
The use of base_path, which is defined in a different file may not be a good idea for future development and the correct understanding of the files.

Reply:
Change is undone in https://github.com/wazuh/wazuh-packages/commit/72c57a91334163a9e7b7bc242d77f5a857fc31ae.
Also created a related issue: https://github.com/wazuh/wazuh-packages/issues/2092